.taskcluster.yml

```diff
@@ -18,21 +18,20 @@ tasks:
 
     owner: ${event.sender.login}@users.noreply.github.com
 
-    shouldPush:
+    push_docker_image:
       $if: 'tasks_for == "github-pull-request"'
-      then: false
+      then: '0'
```

Comment:
Coming from reviewing a bash script, I almost said this is reversed :)

Reply:
I've just unified this with the approach we took in scriptworker-scripts.